lms/services/assignment.py

```diff
@@ -149,6 +150,9 @@ def set_document_url(
                 ext_lti_assignment_id=ext_lti_assignment_id,
             )
 
+        if extra:
+            assignment.extra = extra
+
```

Comment:
We want to store some info we see even if we don't query it directly, more details about canva's file, group set id ...